doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -382,6 +382,8 @@ nRF Desktop
     This ensures correct behavior if another source requests Constant Latency sub-power mode through the nrfx API.
   * The :ref:`CONFIG_DESKTOP_CONSTLAT_DISABLE_ON_STANDBY <config_desktop_app_options>` Kconfig option depends on :kconfig:option:`CONFIG_CAF_PM_EVENTS`.
     CAF Power Management events support is necessary to disable constant latency interrupts on standby.
+  * The main application's configuration directory (``APPLICATION_CONFIG_DIR`` CMake option) is set only in the :file:`CMakeLists.txt` file of the application.
```

Comment:
perhaps explain purpose of this chage?

(Copying project contents to different directory maintains correct CMake configuration)

Reply:
The problematic approach was not used in NCS 2.7 (the approach was introduced by 1af3bec0cee2 after NCS 2.7 was released). We describe changes introduced since NCS 2.7, so I made the note more generic.

Reply:
let's remove the note

Reply:
Removed note from the PR (as requested by @divipillai). I still wonder if we shouldn't keep the note in general.

Reply:
Discussed with @kapi-no during standup. We agreed not to add the release note here.